types/k6/experimental/browser.d.ts

```diff
@@ -0,0 +1,1508 @@
+       * height of clipping area
+       */
+      height: number;
+    };
+
+    /**
+     * When true, takes a screenshot of the full scrollable page, instead of
+     * the currently visible viewport. Defaults to `false`.
+     */
+    fullPage?: boolean;
+
+    /**
+     * Hides default white background and allows capturing screenshots with
+     * transparency. Not applicable to `jpeg` images. Defaults to `false`.
+     */
+    omitBackground?: boolean;
+
+    /**
+     * The file path to save the image to. The screenshot type will be inferred
+     * from file extension. If `path` is a relative path, then it is resolved
+     * relative to the current working directory. If no path is provided, the
+     * image won't be saved to the disk.
+     */
+    path?: string;
+
+    /**
+     * The quality of the image, between 0-100; `jpeg` only.
+     */
+    quality?: number;
+
+    /**
+     * Specify screenshot type, defaults to `png`.
+     */
+    type?: "png"|"jpeg";
+   }): ArrayBuffer;
+
+  /**
+   * **NOTE** Use locator-based locator.selectOption(values[, options]) instead.
+   *
+   * This select one or more options which match the values from a <select>
+   * element.
+   *
+   * @param selector A selector to search for an element. If there are multiple
+   * elements satisfying the selector, the first will be used.
+   * @param values Options to select. If the select has multiple attribute, all
+   * matching options are selected, otherwise only the first option matching
+   * one of the passed options is selected. Object can be made up of keys with
+   * value, label or index.
+   * @param options
+   */
+  selectOption(selector: string, values: string|ElementHandle|SelectOptionsObject|string[]|ElementHandle[]|SelectOptionsObject[], options?: {
+    /**
+     * Setting this to `true` will bypass the actionability checks (visible,
+     * stable, enabled). Defaults to `false`.
+     */
+    force?: boolean;
+
+    /**
+     * If set to `true` and a navigation occurs from performing this action, it
+     * will not wait for it to complete. Defaults to `false`.
+     */
+    noWaitAfter?: boolean;
+
+    /**
+     * When `true`, the call requires selector to resolve to a single element.
+     * If given selector resolves to more than one element, the call throws
+     * an exception. Defaults to `false`.
+     */
+    strict?: boolean;
+
+    /**
+     * Maximum time in milliseconds. Defaults to `0` - no timeout. Default is
+     * overridden by the `setDefaultTimeout` option on `BrowserContext` or
+     * `page` methods.
+     */
+    timeout?: number;
+  }): string[];
+
+  /**
+   * Set the supplied html string to the current page.
+   *
+   * @param html HTML markup to assign to the page.
+   * @param options
+   */
+  setContent(html: string, options?: {
+    /**
+     * Maximum operation time in milliseconds. Defaults to `0` - no timeout. The
+     * default value can be changed via `navigationTimeout` option in the config,
```

Comment:
I believe this `navigationTimeout` option in the config does not apply to our implementation right?